client/extensions/woocommerce/app/order/order-customer-info.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+
+class OrderCustomerInfo extends Component {
+	render() {
+		const { order, translate } = this.props;
+		if ( ! order ) {
+			return null;
+		}
+
+		const { billing, shipping } = order;
+
+		return (
+			<div className="order__customer-info">
+				<SectionHeader label={ translate( 'Customer Information' ) } />
+				<Card>
+					<h3 className="order__billing-details">{ translate( 'Billing Details' ) }</h3>
+					<div className="order__customer-billing">
+						<h4>{ translate( 'Address' ) }</h4>
+						<div className="order__billing-address">
```

Comment:
It would be nice to use the `AddressView` component for this since it has a static mode. It would make later (after v1) implementation of editing easier too. However, the component currently doesn't support all address fields (like`address_2`) -- this PR would need merged first https://github.com/Automattic/wp-calypso/pull/15321/files#diff-c93c30ca2f4962460ce0983daa5b88b5. I would be fine with just a `TODO` for the future so we remember so you don't get held up.